crates/astria-conductor/local.env.example

```diff
@@ -33,6 +33,12 @@ ASTRIA_CONDUCTOR_SEQUENCER_URL="ws://127.0.0.1:26657/websocket"
 # the sequencer block height that the rollup's first block was in
 ASTRIA_CONDUCTOR_INITIAL_SEQUENCER_BLOCK_HEIGHT=1
 
+# the data availability block height that the rollup's first block was in
+ASTRIA_CONDUCTOR_INITIAL_DA_BLOCK_HEIGHT=1
+
+# The number of block on DA in which the first sequencer block for the rollup should be found
+ASTRIA_CONDUCTOR_DA_BLOCK_RANGE=100
```

Comment:
I renamed this to `ASTRIA_CONDUCTOR_CELESTIA_SEARCH_WINDOW`, calling it a range was a little misleading.
The search window is the number of blocks on Celestia where the next sequencer block must appear, otherwise the chain cannot continue.
The initial block height is the block number on da where the first sequencer block for the rollup should be.